clients/fides-js/src/components/Overlay.tsx

```diff
@@ -46,26 +55,41 @@ const Overlay: FunctionComponent<Props> = ({
   const hasMounted = useHasMounted();
   const [bannerIsOpen, setBannerIsOpen] = useState(false);
 
+  const dispatchCloseEvent = () => {
```

Comment:
we may want to put this in a `useCallback` since it is relied on by a `useCallback` (`handleOpenModal`)

Reply:
@allisonking I just updated this- let me know if this is correct!

Reply:
yes, that looks right to me, nice job!